downstream/modules/platform/ref-controller-system-requirements.adoc

```diff
@@ -0,0 +1,94 @@
+[id="ref-controller-system-requirements"]
+
+== {ControllerNameStart} system requirements
+
+{ControllerNameStart} is a distributed system, where different software components can be co-located or deployed across multiple compute nodes.
+In the installer, node types of control, hybrid, execution, and hop are provided as abstractions to help you design the topology appropriate for your use case.
+
+The following tables provides recommendations for node sizing:
+
+[NOTE]
+====
+On all nodes except hop nodes, allocate a minimum of 20 GB to `/var/lib/awx` for execution environment storage.
+====
+
+.Execution nodes
+Runs automation. Increases memory and CPU to increase capacity for running more forks
+
+[cols="a,a",options="header"]
+|===
+h| Requirement | Required
+| *RAM* | 16 GB  
+| *CPUs* | 4
+|===
+
+.Control nodes 
+Processes events and runs cluster jobs including project updates and cleanup jobs. 
+Increasing CPU and memory can help with job event processing.
+
+
+|===
+h| Requirement | Required
+| *RAM* | 16 GB  
+| *CPUs* | 4
+|===
+
+.Hybrid nodes
+Runs both automation and cluster jobs, comments for both execution and control nodes apply to this node type.
```

Comment:
Does "comments" means "anything in this guide that talks about execution and control nodes"? I am not sure readers will all catch this. Not